geotorch/constraints.py

```diff
@@ -550,3 +550,28 @@ def positive_semidefinite_fixed_rank(
             a custom callable. Default: ``"expm"``
     """
     return _register_manifold(module, tensor_name, PSSDFixedRank, rank, f, triv)
+
+
+def alpha_stable(module: torch.nn.Module, tensor_name="weight", alpha=1e-3):
+    r"""Adds an alpha_stability parametrization to the matrix ``module.tensor_name``.
+
+    When accessing ``module.tensor_name``, the module will return the parametrized
+    version :math:`A` so that :math:`\min_{\lambda \in Sp(A)} \Re(\lambda)<= -\alpha`.
+
+    If the tensor has more than two dimensions, the parametrization will be
+    applied to the last two dimensions.
+
+    Examples::
+
+        >>> layer = nn.Linear(30, 30)
+        >>> alpha_stable(layer, "weight", alpha=2)
+        >>> torch.all(torch.linalg.eigvals(layer.weight)<=-layer.alpha)
+        True
+
+    Args:
+        module (nn.Module): module on which to register the parametrization
+        tensor_name (string): name of the parameter, buffer, or parametrization
+            on which the parametrization will be applied. Default: ``"weight"``
+        alpha (float): Bound on the decay rate and eigevenalues of matrix A
+    """
+    return _register_manifold(module, tensor_name, Hurwitz, alpha)
```

Comment:
Add an end of line (the linter should have fixed this, but it's not well maintained...)